client/app/components/Modal/Modal.scss

```diff
@@ -34,7 +34,8 @@
     z-index: $z-index-front;
     @media screen and (max-width: $small) {
       width: 100vw;
-      height: 100vh;
+      height: 100%;
+      margin: 0 auto;
```

Comment:
Is this line necessary?

Reply:
Margin: 0 auto has been removed as well.